pkg/ebpf/tracer.go

```diff
@@ -166,6 +170,16 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars setups up endpoint 8080 as the debug http endpoint and exposes some metrics for tracer
+func (t *Tracer) setupExpvars() {
+	go http.ListenAndServe(":8080", nil)
+
+	expvar.Publish("tracer.perf_received", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.perfReceived) }))
+	expvar.Publish("tracer.perf_lost", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.perfLost) }))
+	expvar.Publish("tracer.skipped_conns", expvar.Func(func() interface{} { atomic.LoadUint64(&t.skippedConns) }))
+	expvar.Publish("tracer.expired_tcp_conns", expvar.Func(func() interface{} { atomic.LoadUint64(&t.expiredTCPConns) }))
```

Comment:
I think it would be nice to also publish the `state` counters: https://github.com/DataDog/datadog-agent/blob/master/pkg/ebpf/state.go#L392-L400

Reply:
(some of them are already published though)